web_src/js/features/package-editor.js

```diff
@@ -0,0 +1,31 @@
+import $ from 'jquery';
+
+const {csrfToken} = window.config;
+
+function initEditPreviewTab($form) {
+  const $tabMenu = $form.find('.tabular.menu');
+  $tabMenu.find('.item').tab();
+  $tabMenu.find(`.item[data-tab="${$tabMenu.data('preview')}"]`).on('click', function () {
```

Comment:
This selector looks **really** fragile.
Please create a specific class instead and query for that class.

Reply:
That's just the old code, after the "editor" refactoring, no such code is needed.

Actually, this PR has been stale for too long time, I guess the author might have lost interest on it?


If it is still stale and inactive, it needs the maintainers who have writer permission to help.